src/bin/cargo/commands/verify_project.rs

```diff
@@ -5,12 +5,16 @@ use std::process;
 
 pub fn cli() -> Command {
     subcommand("verify-project")
-        .about("Check correctness of crate manifest")
+        .about(color_print::cstr!(
+            "\
+Check correctness of crate manifest.
+
+Deprecated, use `<cyan,bold>cargo locate-project --workspace</>` instead (also checks UTF8).\
```

Comment:
I would avoid mentioning `cargo locate-project`. Same as `cargo verify-project`, it basically checks only TOML validity. The purpose of `cargo locate-project` is not checking manifest either. This is not like the relationship of cargo-read-manifest to cargo-metadata.

I would probably put this instead:

```
Deprecated, see https://github.com/rust-lang/cargo/issues/14679
```